### ISSUE TITLE
the coder of batch normalization in keras

### ISSUE THREAD

**Issue by hanyaqian** [lvapeab/nmt-keras#25]
Hi @lvapeab 

When self.mode is equal to ['0' or '2'](https://github.com/MarcBS/keras/blob/master/keras/layers/normalization.py#L211) ,training a batch-normalized network, and when self.mode is equal to ['1'](https://github.com/MarcBS/keras/blob/master/keras/layers/normalization.py#L276) ,implementing batch normalizing transform, just as described in the [paper](https://arxiv.org/abs/1502.03167). correct?

Regards

**Comment by hanyaqian** [comment]
And i find it is difficult to debug and check the intermediate results, do you have any good ideas?

Thanks!

**Comment by lvapeab** [comment]
Hi,
Yes, batch norm = 1 is the transform from the paper.

For debugging the code, maybe you should take a look to

**Comment by hanyaqian** [comment]
Thanks for your reply.
